docs/libcurl/curl_url_get.3

```diff
@@ -84,7 +84,14 @@ If the host name is a numeric IPv6 address, this field might also be set.
 .IP CURLUPART_PORT
 Port cannot be URL decoded on get.
 .IP CURLUPART_PATH
+\fIpart\fP will always start with a `/' even with a zero-length path.
 .IP CURLUPART_QUERY
+The initial question mark that denotes the begin of the query will not be a
+part of the returned query.
```

Comment:
How about:

The initial question mark that denotes the beginning of the query part is a delimiter only. It is not part of the query contents.

Reply:
LGTM